docs/tempo/website/grafana-agent/tail-based-sampling.md

```diff
@@ -47,10 +52,15 @@ tempo:
       ...
       tail_sampling:
         policies:
```

Comment:
Do we want to link to the upstream docs?
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/tailsamplingprocessor/README.md
Especially knowing how `decision_wait` and `num_traces` work is be valuable. But maybe we can add a section for that as well here?

Reply:
All these configs are already well documented in the Agent config. My concern is having the same information in two places, as it may lead to outdated information very quickly. I think that users will reach the agent config docs if they want further details, but I could be wrong 🤷‍♂️